src/main/java/com/amazonaws/services/simpleworkflow/flow/WorkflowContext.java

```diff
@@ -60,15 +60,25 @@
      * This method is created to enable workflow updates without changing their
      * type version. The code changes should follow the following pattern:
      * <p>
-     * 
+     *
      * <pre>
+     <<<<<<< HEAD
      * if (workflowContext.isImplementationVersion(COMPONENT_NAME, 5)) {
      *     // New code path
      * }
      * else if (workflowContext.isImplementationVersion(COMPONENT_NAME, 4)) {
      *     // Old version 4 code path
      * }
      * else if (workflowContext.isImplementationVersion(COMPONENT_NAME, 3)) {
+     =======
```

Comment:
Probably during conflict resolution you forgot to fix this. Please fix it.

Reply:
Fixed.